src/graph/GraphViewServer.ts

```diff
@@ -282,6 +284,7 @@ export class GraphViewServer extends EventEmitter {
     return message;
   }
 
+  // tslint:disable-next-line:no-any
   private async executeQuery(queryId: number, gremlinQuery: string): Promise<any[]> {
```

Comment:
nit: I don't really care if you use `any[]` or `Object[]`, but I think `executeQuery`, `_executeQueryCore`, and `_executeQueryCoreForEndpoint` should all have the same return type

Reply:
If I had to choose, I'd probably pick `any[]` and just disable "no-any" for this file. I don't think `Object[]` is helping us at all and you end up casting it to `any[]` eventually anyways